homeassistant/components/vallox/config_flow.py

```diff
@@ -0,0 +1,131 @@
+"""Config flow for the Vallox integration."""
+from __future__ import annotations
+
+import ipaddress
+import logging
+from typing import Any
+
+from vallox_websocket_api import Vallox
+from vallox_websocket_api.exceptions import ValloxApiException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+    }
+)
+
+VALLOX_CONNECTION_EXCEPTIONS = (
+    OSError,
+    ValloxApiException,
+)
+
+
+def ip_valid(host: str) -> bool:
+    """Return True if the host is a valid IP address."""
+    try:
+        if ipaddress.ip_address(host).version in [4, 6]:
+            return True
+    except ValueError:
+        pass
+
+    return False
+
```

Comment:
```suggestion
```

You can use the built-in util intead
`from homeassistant.util.network import is_ip_address`

Reply:
Thanks, that's better. 👍 Fixed.